pkg/pillar/types/contenttreetypes.go

```diff
@@ -29,6 +29,8 @@ type ContentTreeConfig struct {
 	GenerationCounter int64
 	DisplayName       string
 	CustomMeta        string
+	// Do we download on this node ?
+	IsLocal bool
```

Comment:
does local mean we want to download?

Reply:
Yes if Local is true the content tree will be downloaded on that node.